src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/LoggingConstants.cs

```diff
@@ -13,5 +13,13 @@ internal class LoggingConstants
         public const string RefreshFailedDueToKeyVaultError = "A refresh operation failed while resolving a Key Vault reference.";
         public const string RefreshFailedError = "A refresh operation failed.";
         public const string RefreshCanceledError = "A refresh operation was canceled.";
+
+        // Successful update messages
+        public const string SuccessfulConfigurationUpdated = "IConfiguration updated.";
```

Comment:
>IConfiguration updated. [](http://example.com/codeflow?start=62&length=23)

To make it more explicit that the entire configuration changed, we can say something like:
"IConfiguration updated from Azure App Configuration endpoint: "

We can follow the same pattern for all messages.